sdk/core/core/src/az_json_pointer.c

```diff
@@ -67,11 +67,14 @@ AZ_NODISCARD static az_result az_span_reader_read_json_pointer_char(az_span* sel
   }
 }
 
+/**
+ * Returns a next reference token in the JSON pointer. The JSON pointer parser is @var
```

Comment:
What do you mean by `The JSON pointer parser is @var az_span_reader.`?
Also, what's the purpose of `@var`?

Reply:
This text is just bad and should be fixed; it refers to an az_span_reader which we don't have any more - this is not public docs.